projects/step-core/src/lib/components/autorefresh-toggle/autorefresh-toggle.component.scss

```diff
@@ -15,3 +15,11 @@
     border-radius: 6px;
   }
 }
+
+.grey-border {
+  border: 1px solid rgb(208, 213, 221);
+}
+
+.z-1000 {
+  z-index: 1000;
+}
```

Comment:
Avoid one-rule only classes. It's better to define something like `.refresh-btn` and group them.
If this border color still required (check the possibility to apply `mat-raised-button` first) take the color from `.core-variables.scss`

Reply:
Added refresh-btn class